README.md

```diff
@@ -25,6 +25,10 @@ To install:
 
     $ sudo pip install scuba
 
+To install with `argcomplete` (for [bash completion support](#bash-completion)):
+
+    $ sudo pip install scuba[ARGCOMPLETE]
```

Comment:
Shouldn't this be lowercase, to match the name in `extras_require` from `setup.py`?
```
    $ sudo pip install scuba[argcomplete]
```

Reply:
After testing both `pip install .[ARGCOMPLETE]` and `pip install .[argcomplete]`, it appears the "extras" are actually case-insensitive. However, I still think lowercase is correct.

Reply:
Good point.  I'll fix it.

Reply:
Addressed by TimothySprague/scuba@ea1be7f6fb2a25a586329f831373ff28e7a0a2db